packages/online-playground/src/App.vue

```diff
@@ -19,15 +19,47 @@ if (hash.startsWith('__DEV__')) {
   useDevMode.value = true
 }
 
-// TODO: we should fetch the latest version and set it by default here
-const store = new ReplStore({
-  serializedState: hash,
-  defaultVueRuntimeURL:
-    'https://cdn.jsdelivr.net/npm/@vue/runtime-dom@3.4.21/dist/runtime-dom.esm-browser.js',
-  defaultVueServerRendererURL:
-    'https://cdn.jsdelivr.net/npm/@vue/server-renderer@3.4.21/dist/server-renderer.esm-browser.js',
+const {
+  importMap: builtinImportMap,
+  vueVersion,
+  productionMode,
+} = useVueImportMap({
+  runtimeDev: 'https://cdn.jsdelivr.net/npm/vue@3/dist/vue.runtime.esm-browser.js',
```

Comment:
We actually really need to fetch the latest version from npm and use that specific tag. We cannot use `@3` as a default because the idea is that a link should always work or fail the same way